### DISCUSSION TITLE
Is there any way to turn the tracker off?

### DISCUSSION THREAD

**Discussion by goodlifer** [ActivityWatch/activitywatch#933] [Q&A]
The title says it all

**Comment by BelKed** [comment]
If you want to turn ActivityWatch off completely, simply close the ActivityWatch application.

---

If you just want to turn off a particular watcher, you'll need to find the ActivityWatch icon in your system tray, and from there you can turn the watcher on and off:

<p align=center>
<img width=500px src="https://github.com/ActivityWatch/activitywatch/assets/66956532/36faad57-2bef-4b83-b59e-7c89f538e46d">
</p>

---

Feel free to ask if you have specific questions about disabling the tracker on your operating system :)
